base/uk.ac.stfc.isis.ibex.ui.journalviewer/src/uk/ac/stfc/isis/ibex/ui/journalviewer/JournalViewerView.java

```diff
@@ -245,7 +247,7 @@ private void changeTableColumns() {
         for (final JournalField field : JournalField.values()) {
             if (model.getFieldSelected(field)) {
                 TableViewerColumn col = table.createColumn(field.getFriendlyName(), 1, true,
-                        new DataboundCellLabelProvider<JournalRow>(table.observeProperty("row")) {
+                        new DataboundCellLabelProvider<JournalRow>(new ObservableMap(Collections.emptyMap())) {
```

Comment:
What was the purpose of this? It seems unrelated to the ticket?

Reply:
The JournalViewer table uses an extension of the DataboundTable class. The changes to the underlying table have cause more changes in the JournalViewer in subsequent updates to this branch, so this is no longer an issue.